seahorn/lib/bounds.c

```diff
@@ -3,7 +3,7 @@
 
 size_t sea_max_buffer_size(void) { return MAX_BUFFER_SIZE; }
 size_t sea_max_string_len(void) { return MAX_STRING_LEN; }
-size_t sea_max_array_list_initial_size(void) { return MAX_INITIAL_SIZE; }
+size_t sea_max_array_list_initial_size(void) { return KLEE_MAX_SIZE; }
```

Comment:
either the name of the function is wrong, or it's value. It is called `initial_size`, but returns `KLEE_MAX_SIZE`